runtime/compiler/control/CompilationThread.cpp

```diff
@@ -2859,6 +2859,14 @@ void TR::CompilationInfo::stopCompilationThreads()
       fprintf(stderr, "numProfiledMethodGuardsValidationFailed: %d\n", aotStats->profiledMethodGuards.numFailedValidations);
       fprintf(stderr, "numProfiledMethodGuardsValidationSucceeded: %d\n", aotStats->profiledMethodGuards.numSucceededValidations);
       fprintf(stderr, "-------------------------\n");
+
+      fprintf(stderr, "RELO FAILURES BY TYPE ------\n");
+      for (uint32_t i = 0; i < TR_NumExternalRelocationKinds; i++)
+         {
+         fprintf(stderr, "%s: %d\n", TR::ExternalRelocation::getName((TR_ExternalRelocationTargetKind)i), aotStats->numRelocationsFailedByType[i]);
```

Comment:
nit: shouldn't that be `%u` ?

Reply:
Heh, yeah, given that I'm iterating using a `uint32_t`. I can change it if you'd like, or perhaps maybe just change it as part of a future cleanup PR.